test/parallel/test-process-beforeexit.js

```diff
@@ -55,6 +55,14 @@ function tryRepeatedTimer() {
   const repeatedTimer = common.mustCall(function() {
     if (++n < N)
       setTimeout(repeatedTimer, 1);
+    else // n == N
+      process.once('beforeExit', common.mustCall(tryNextTick));
```

Comment:
Perhaps the comment on lines 49-51 should be updated to indicate that this function now does this additional bit of testing.